src/vs/workbench/services/search/node/rawSearchService.ts

```diff
@@ -417,6 +417,9 @@ const FileMatchItemAccessor = new class implements IItemAccessor<IRawFileMatch>
 	}
 
 	getItemPath(match: IRawFileMatch): string {
+		if (match.base) {
```

Comment:
In what cases is base not defined? Won't this function return the absolute path in most cases? If so, it might be too drastic of a change to compare using absolute paths most of the time.